pegasus/sites.v3/code.org/public/css/csc-module.scss

```diff
@@ -0,0 +1,59 @@
+$color-black: black;
+$color-white: white;
+
+.csc-module {
+  padding: 1em;
+
+  .csc-module-outer-wrapper {
+    border: 1px solid;
+    border-radius: 8px;
+  }
+
+  a {
+    font-family: 'Gotham 4r', sans-serif;
+    color: $color-black !important;
```

Comment:
why do we need important here and below? (we definitely might just curious why!)

Reply:
I think this was leftover from my earlier refactor process — just tried it without and we don't need them! ✨